### PR TITLE
e2e: add multiple labels to node

### LINKED ISSUE #4146: e2e: enhance `CreateNodeLabel()` to take multiple labels
# Describe the enhancement #
 > lets enhance CreateNodeLabel to take map[string]string and set the label once (not as part of this PR)

_Originally posted by @Madhu-1 in https://github.com/ceph/ceph-csi/pull/4111#discussion_r1331518334_

### PR DESCRIPTION
**Pull request by Madhu-1**
update the e2e code to add multiple labels to the node at a time.

fixes: #4146

### CODE REVIEW

**Comment by iPraveenParihar** [comment]
Can we do the same for `deleteNodeLabel()` ?

**Comment by Madhu-1** [comment]
> Can we do the same for `deleteNodeLabel()` ?

i didnt noticed that earlier, Done same, PTAL

File: e2e/node.go [hunk 1]
```diff
@@ -26,29 +26,38 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+	testutils "k8s.io/kubernetes/test/utils"
 )
 
-func createNodeLabel(f *framework.Framework, labelKey, labelValue string) error {
+func addLabelsToNodes(f *framework.Framework, labels map[string]string) error {
```
Comment (iPraveenParihar) [inline e2e/node.go:32]:
```suggestion
func addNodeLabels(f *framework.Framework, labels map[string]string) error {
```

**Review by iPraveenParihar — COMMENTED** [review]
small nits

File: e2e/node.go (hunk 1 above)
Comment (Madhu-1) [reply e2e/node.go:32]: addLabelsToNodes seems more opt to me, same is used in Kuberentes util as well.

Comment (iPraveenParihar) [reply e2e/node.go:32]: thought using `deleteNodeLabels` and `addNodeLabels` would be a good idea.

Comment (Madhu-1) [reply e2e/node.go:32]: followed the format from the kubernetes package as an example, lets see what others think, based on the will change this if required.

**Review by nixpanic — APPROVED** [review]

**Review by iPraveenParihar — APPROVED** [review]

**Review by riya-singhal31 — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio rebase

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by nixpanic** [comment]
> /retest ci/centos/mini-e2e-helm/k8s-1.27

not sure why it failed, e2e seems to report a panic [logs](https://jenkins-ceph-csi.apps.ocp.cloud.ci.centos.org/job/mini-e2e-helm_k8s-1.27/508/display/redirect)

**Comment by nixpanic** [comment]
@mergifyio requeue

**Pull request merged**